recipes/islpy/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "islpy" %}
+{% set version = "2016.2.1" %}
+{% set sha256 = "b23ccca253945ba7b41d091e86b3e0ace6a256238f28abf604de1d06f878895c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt && cd test && python -m pytest
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - pytest
+    - cffi
+  run:
+    - python
+    - cffi
+    - six
+
+test:
+  requires:
+    - pytest
+  imports:
+    - islpy
+
+  # Tests run as part of build hold until conda-build 2.0 is in use on conda forge.
+  # https://github.com/conda-forge/staged-recipes/pull/1478
+  #
+  # source_files:
```

Comment:
Can do something like [this](https://github.com/conda-forge/fiona-feedstock/blob/master/recipe/meta.yaml#L37-L48) for now?